andela/ah-the-answer-frontend
# #164829341 Adds functionality to list all bookmarks

#### What does this PR do?
Adds functionality to list all bookmarks for an article

#### Description of the task to be completed
- [x] List all bookmarks

#### How should this be manually tested
- Clone the repo and navigate to the root directory using a terminal
- Install all the dependencies by typing 
```javascript
npm install
```
- Start the server by typing 
```javascript
npm start
```
- Navigate to the main page using a browser at `http://localhost:3000/bookmarks`
- Sign up and log in
- Once on the homepage click an article to view its details
- Click on the bookmark button to bookmark
- Toggle the bookmark button to remove it
- Go to profile page to see the five most recent bookmarks
<img width="1438" alt="Screenshot 2019-05-20 at 11 58 32" src="https://user-images.githubusercontent.com/8179091/58009244-ae84e800-7af6-11e9-83b5-4f846bf1bde1.png">
<img width="1440" alt="Screenshot 2019-05-20 at 11 58 45" src="https://user-images.githubusercontent.com/8179091/58009248-b04eab80-7af6-11e9-9c91-3aefa4bbb859.png">

#### Relevant PT stories
[164829341](https://www.pivotaltracker.com/story/show/164829341)
